gradle.properties

```diff
@@ -1,6 +1,6 @@
 version=0.0.0-development
 group=io.littlehorse
-kafkaVersion=3.8.0
+kafkaVersion=3.9.1
```

Comment:
I would wait until 3.9.1 was released before merging this PR :)

Reply:
When Dr. Streams himself says to wait, I will wait!

However, by the time 3.9.1 is released, we will probably have finished the 4.0 testing already. We are waiting for Stirmzi 0.46 so that we can stress-test the 4.0 Streams client against the 4.0 brokers.

In particular, KIP-890 changes the transactions protocol, which will definitely affect LittleHorse. 890 isn't enabled by default in 3.9.x so I felt it was safe to upgrade without extensive testing, especially given that there are some known issues in 3.8.x that affect LH and are scheduled to be fixed in 3.9.1.

But I am eager for 4.0, as it fixes [KAFKA-17455](https://issues.apache.org/jira/browse/KAFKA-17455), which is the biggest issue affecting us.

Reply:
Actually, it looks like you backported `17455` to the `3.9.x` branch. Thanks so much @mjsax ! This is a huge help. As I said I'm nervous about 4.0 due to KIP-890, so this will buy us time to do more testing before the upgrade.